test/testsets.json

```diff
@@ -52,6 +52,7 @@
     { "name": "test_module_json.js" },
     { "name": "test_module_require.js", "skip": ["nuttx"], "reason": "not implemented for nuttx" },
     { "name": "test_module_dynamicload.js", "skip": ["darwin", "nuttx", "tizenrt"], "reason": "embedded and macos does not support dynamic loading" },
+    { "name": "test_mqtt.js", "skip": ["all"], "reason": "MQTT module is not enabled by default" },
```

Comment:
AFAIK, there is no reason to add the skip entry. The test framework should detect if a module is enabled or not and report it accordingly. Or did something changed?